### PR TITLE
Add audit log events for hotbackup.

### PR DESCRIPTION
**Pull request by neunhoef**

### CODE REVIEW

**Comment by neunhoef** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/8177/

**Comment by neunhoef** [comment]
See also PRs https://github.com/arangodb/enterprise/pull/382 and https://github.com/arangodb/docs/pull/309

File: arangod/Cluster/ClusterMethods.cpp [hunk 1]
```diff
@@ -3724,13 +3742,15 @@ arangodb::Result hotBackupCoordinator(ClusterFeature& feature, VPackSlice const
       result.reset(TRI_ERROR_HOT_BACKUP_INTERNAL,
                    std::string("failed to acquire agency dump: ") + result.errorMessage());
       LOG_TOPIC("c014d", ERR, Logger::BACKUP) << result.errorMessage();
+      events::CreateHotbackup(backupId, TRI_ERROR_HOT_BACKUP_INTERNAL);
```
Comment (jsteemann) [inline arangod/Cluster/ClusterMethods.cpp]:
```suggestion
      events::CreateHotbackup(timeStamp + "_" + backupId, TRI_ERROR_HOT_BACKUP_INTERNAL);
```

File: arangod/Cluster/ClusterMethods.cpp [hunk 2]
```diff
@@ -3885,9 +3914,11 @@ arangodb::Result hotBackupCoordinator(ClusterFeature& feature, VPackSlice const
       }
     }
 
+    events::CreateHotbackup(timeStamp + "_" + backupId, result.errorNumber());
```
Comment (jsteemann) [inline arangod/Cluster/ClusterMethods.cpp]:
```suggestion
    events::CreateHotbackup(timeStamp + "_" + backupId, TRI_ERROR_NO_ERROR);
```

**Comment by neunhoef** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/8216/

File: arangod/Cluster/ClusterMethods.cpp (hunk 2 above)
Comment (neunhoef) [reply arangod/Cluster/ClusterMethods.cpp]: Done

File: arangod/Cluster/ClusterMethods.cpp (hunk 1 above)
Comment (neunhoef) [reply arangod/Cluster/ClusterMethods.cpp]: Done.

**Comment by KVS85** [comment]
Tests blue except known unrelated failures.

**Pull request merged**